.eslintrc

```diff
@@ -14,14 +14,16 @@
     "plugin:cypress/recommended"
   ],
   "parser": "@typescript-eslint/parser",
-  "plugins": ["react", "react-hooks", "@typescript-eslint", "prettier", "import"],
+  "plugins": ["react", "react-hooks", "@typescript-eslint", "prettier", "import", "unused-imports"],
   "rules": {
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
     "react/prop-types": "off",
     "prettier/prettier": "warn",
+    "unused-imports/no-unused-imports": "error",
     "@typescript-eslint/explicit-module-boundary-types": "off",
-    "@typescript-eslint/ban-ts-comment": "warn"
+    "@typescript-eslint/ban-ts-comment": "warn",
+    "@typescript-eslint/no-unused-vars": "off"
```

Comment:
Does this need to be switched off for a reason?

Reply:
I referred this documentation here also it said that it might report incorrect errors

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unused-vars.md